src/genesis.rs

```diff
@@ -392,6 +394,24 @@ impl Genesis {
         (
             map,
             vec![
+                Transaction::wrap(TransactionData::RegisterContract(
+                    RegisterContractTransaction {
+                        owner: "hyle".into(),
+                        verifier: "native".into(),
+                        program_id: ProgramId("blst".as_bytes().to_vec()),
+                        state_digest: StateDigest(vec![]),
+                        contract_name: "blst".into(),
+                    },
+                )),
+                Transaction::wrap(TransactionData::RegisterContract(
+                    RegisterContractTransaction {
+                        owner: "hyle".into(),
+                        verifier: "native".into(),
+                        program_id: ProgramId("sha3_256".as_bytes().to_vec()),
+                        state_digest: StateDigest(vec![]),
+                        contract_name: "sha3_256".into(),
+                    },
+                )),
```

Comment:
I think it would make more sense to have the `verifier` field be `sha3_256` with possibly some extra parametrisation metadata in program_id. Right now we effectively hardcode that some contract names are "magical", even if they end up not being registered on the node at some point.

If we embed this in the verifier you can "skip" those by just not registering the contracts, and the state is more consistent.